Sources/TuistAutomation/Utilities/BuildGraphInspector.swift

```diff
@@ -7,7 +7,7 @@ public protocol BuildGraphInspecting {
     /// Returns the build arguments to be used with the given target.
     /// - Parameter target: Target whose build arguments will be returned.
     /// - Parameter configuration: The configuration to be built. When nil, it defaults to the configuration specified in the scheme.
```

Comment:
I'd also add documentation for `skipSigning` (eg. you can add a reason for why it might make sense to skip signing)